.github/workflows/release.yml

```diff
@@ -0,0 +1,108 @@
+
+name: Release
+
+on:
+  # Only run the workflow when a PR is closed, or when a developer explicitly requests
+  # a build by sending a 'firebase_build' event.
+  pull_request:
+    types:
+      - closed
+
+  repository_dispatch:
+    types:
+      - firebase_build
+
+jobs:
+  stage_release:
+    # If triggered by a PR it must be merged and contain the label 'release:build'.
+    if: github.event.action == 'firebase_build' ||
+      (github.event.pull_request.merged &&
+      contains(github.event.pull_request.labels.*.name, 'release:build'))
+
+    runs-on: ubuntu-latest
+
+    # When manually triggering the build, the requester can specify a target branch or a tag
+    # via the 'ref' client parameter.
+    steps:
+    - name: Checkout source for staging
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.client_payload.ref || github.ref }}
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+
+    - name: Run tests
+      run: |
+        pytest
+        echo "Running integration tests"
+
+    - name: Package release artifacts
+      run: python setup.py bdist_wheel bdist_egg
+
+    # Attach the packaged artifacts to the workflow output. These can be manually
+    # downloaded for later inspection if necessary.
+    - name: Archive artifacts
+      uses: actions/upload-artifact@v1
+      with:
+        name: dist
+        path: dist
+
+    # Check whether the release should be published. We publish only when the trigger PR is
+    #   1. merged
+    #   2. to the master branch
+    #   3. with the title prefix '[chore] Release '.
+    - name: Publish preflight check
+      if: success() && github.event.pull_request.merged &&
+        github.ref == 'master' &&
+        startsWith(github.event.pull_request.title, '[chore] Release ')
+      id: preflight
+      run: |
+        ./.github/scripts/publish_preflight_check.sh
+        echo ::set-env name=FIREBASE_PUBLISH::true
+
+    # Tag the release if not executing in the dryrun mode. We pull this action froma
+    # custom fork of a contributor until https://github.com/actions/create-release/pull/32
+    # is merged. Also note that v1 of this action does not support the "body" parameter.
+    - name: Create release tag
+      if: success() && env.FIREBASE_PUBLISH
+      uses: fleskesvor/create-release@1a72e235c178bf2ae6c51a8ae36febc24568c5fe
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ steps.preflight.outputs.version }}
+        release_name: Firebase Admin Python SDK ${{ steps.preflight.outputs.version }}
+        body: ${{ steps.preflight.outputs.changelog }}
+        draft: false
+        prerelease: false
+
+    - name: Publish to Pypi
+      if: success() && env.FIREBASE_PUBLISH
+      run: echo Publishing to Pypi
```

Comment:
I assume these actions needs to be implemented in future PRs, right?

Reply:
Yes, that's the plan.